samples/musicfestival-frontend-react/src/App.tsx

```diff
@@ -16,7 +16,7 @@ import { BlockPage } from './pages/BlockPage';
 
 let previousSavedMessage: any = null;
 const singleKeyUrl = process.env.REACT_APP_CONTENT_GRAPH_GATEWAY_URL as string
-const hmacKeyUrl = process.env.REACT_APP_CG_PROXY_URL as string
+const hmacKeyUrl = process.env.REACT_APP_CG_PREVIEW_URL as string
```

Comment:
should rename this variable since it's not the URL to use with HMAC auth anymore, but using preview tokens.
```suggestion
const previewUrl = process.env.REACT_APP_CG_PREVIEW_URL as string
```